### ISSUE TITLE
Switch to REPL without using the mouse

### ISSUE THREAD

**Issue by W1llyw0nk4** [nobody-famous/alive#179]
Let me start by saying that you (guys?) are awesome!

I am relatively new to common lisp, and want to focus on learning the language rather than getting into editors. I understand developing in CL is more powerful if you evaluate code in the REPL, and then poke at it there. 

To that end I've been trying to create a keybinding in vscode that allows me to evaluate a function like so:

"macros.list": {
        "switch-to-lisp-repl": [
            "alive.sendToRepl",
            "lispRepl.focus",
        ]}
But in order to poke at the code, I have to click the REPL panel, specifically the part where my cl-user prompt is. I understand that in emacs and vim there are keybindings that will evaluate the part of the code and take you to the REPL without mouse interaction. 

I would not write you if I had not done my best to figure this one out, but it appears that it simply doesn't exist in vscode. I can switch to the terminal with ctrl+`, I can switch to the lisp repl with the aforementioned "lispRepl.focus", but this leaves the cursor somewhere unknown. I found that quickly pressing tab twice will sometimes bring the cursor to the prompt, but unfortunately this is unreliable.

Am I missing something, because I would love to be able to evaluate code without having to use the mouse. I come from a setup where I switch between editor and command prompt- so I am used to alt+tab a lot when developing. If there is an equivalent in vscode, I would love to know- because I've searched for it a few hours before ending up here.

**Comment by W1llyw0nk4** [comment]
tldr; switch to repl without having to use the mouse- and still be able to type in the repl's prompt

**Comment by W1llyw0nk4** [comment]
tldr; switch to repl without having to use the mouse- and still be able to type in the repl's prompt

**Comment by W1llyw0nk4** [comment]
tldr; switch to repl without having to use the mouse- and still be able to type in the repl's prompt

**Comment by nobody-famous** [comment]
I'm looking into it. Apparently autofocus doesn't work :-/

**Comment by nobody-famous** [comment]
OK, I got a fix for it. I have a list of things I'm trying to go in, but it'll be in the next release. Hopefully in the next couple of weeks.

As a side note, using the scratch pad buffer as an ad-hoc REPL might help since the REPL input can't do completions and whatnot. I mainly use the REPL input to add something to the history so that the REPL history command can be used to quickly rerun it.

**Comment by W1llyw0nk4** [comment]
Thank you so much!
